src/App.js

```diff
@@ -54,31 +55,35 @@ class App extends Component {
     .then(response => response.json())
     .then((data) => {
       this.setState({
-        text: "Your library tracks weere well loaded",
-        tracks: data.items,
+        text: "Your library tracks were well loaded",
+        tracks: data.items !== undefined ? data.items : [],
```

Comment:
If data.items could be empty, initialize it as an empty array instead of checking its emptiness many times